packages/components/src/Typography/css/Emphasis.css

```diff
@@ -3,16 +3,13 @@
 }
 
 .highlight {
-  margin-left: calc(-1 * var(--space-smaller));
-  padding: 0 var(--space-smaller);
-  font-family: var(--typography--fontFamily-display);
+  display: inline;
+  padding: var(--space-smaller) 0;
```

Comment:
Do we actually need this padding? In the first image in the PR description the brushstroke looks closer to the text

Reply:
I checked and we do - it prevents the highlight from breaking into the text at smaller sizes:

**Without**
![image](https://user-images.githubusercontent.com/39704901/210878283-e12da089-86fd-4b31-88f0-f36b7ce8cd07.png)

**With**
![image](https://user-images.githubusercontent.com/39704901/210878428-594d2e8e-397e-4cc1-9f8a-8c2cf753ee4a.png)

Although I wonder if maybe there's a more systematic way of handling it? I think the difference in gap between the smaller size and larger size comes from the "90%" value... Going to try something

Reply:
Update: I tried something and it worked 😄